src/components/nav-bar/nav-bar.tsx

```diff
@@ -0,0 +1,15 @@
+import { Component, h, Host } from "@stencil/core";
+
+@Component({
+  tag: "oui-nav-bar",
+  styleUrl: "nav-bar.css",
+})
+export class NavBar {
+  public render() {
+    return (
+      <Host>
+        <slot></slot>
```

Comment:
On jsx you can self close any tags.

```suggestion
        <slot />
```